ami/models/hifigan_generator.py

```diff
@@ -0,0 +1,175 @@
+# Ref:
+# https://github.com/jik876/hifi-gan/blob/master/models.py
+
+import torch
+from torch.nn import Conv1d, ConvTranspose1d
+
+
+def init_weights(m: torch.nn.Module, mean: float = 0.0, std: float = 0.01) -> None:
+    classname = m.__class__.__name__
+    if classname.find("Conv") != -1:
+        m.weight.data.normal_(mean, std)
+
+
+def get_padding(kernel_size: int, dilation: int = 1) -> int:
+    return int((kernel_size * dilation - dilation) / 2)
```

Comment:
`get_padding`にこの式だけだとどういった目的でpaddingの値を算出するのかわからない。
畳み込みで入出力サイズを変えないようにするため？

```suggestion
def get_padding(kernel_size: int, dilation: int = 1) -> int:
    return int((kernel_size * dilation - dilation) / 2)
```